setup.py

```diff
@@ -14,6 +14,7 @@
         'six',
         'jsonschema',
     ],
+    package_data={'operator_manifest': ['schemas/*.json']},
```

Comment:
Shouldn't be this commit merged with first one?

Reply:
done